src/github/validate.rs

```diff
@@ -169,27 +171,42 @@ fn fetch_pull_request(
                 .try_execute()
                 .chain_err(|| "Failed to fetch pull request commits")?;
 
-        raw_commits
-            .into_iter()
-            .map(|c: types::Commit| {
-                Commit {
-                    sha: c.sha,
-                    author: types::Author {
-                        name: c.commit.author.name,
-                        email: c.commit.author.email,
-                        date: c.commit.author.date,
-                        github_login: Some(c.author.login),
-                    },
-                    committer: types::Author {
-                        name: c.commit.committer.name,
-                        email: c.commit.committer.email,
-                        date: c.commit.committer.date,
-                        github_login: Some(c.committer.login),
-                    },
-                    message: c.commit.message,
+        let mut commits = Vec::new();
+        for c in raw_commits {
+            let (title, description) = {
+                let mut lines = c.commit.message.lines();
+                let title = lines.next().expect("at least one line").to_string();
+                match lines.next() {
+                    Some("") | None => {}
+                    _ => return Err(
```

Comment:
If we want this to be a generic checker, maybe we don't want to assume the title/description format, and instead make that configurable somehow? Like, in order for the title and description fields to be available, you need to set that property in the tailor config.

Reply:
I wasn't certain about this either. My thinking was that this format is pretty widely adopted (I would ubiquitous) so it should be safe.